app/helpers/layout_helper.rb

```diff
@@ -106,10 +106,30 @@ def radio_button_f(f, attr, options = {})
   end
 
   def select_f(f, attr, array, id, method, select_options = {}, html_options = {})
+    array = array.to_a
+    disable_button = select_options.delete(:disable_button)
+    include_blank = select_options.delete(:include_blank)
+
+    array.insert(0, OpenStruct.new({id => '', method => include_blank == true ? nil : include_blank })) if include_blank
```

Comment:
I think this line could be improved in several steps
* `a == true` is probably never good idea, if you want to be explicit, I'd consider `a.is_a?(TrueClass)`
* missing space in `{id`
* I'd wrap the method value in brackets `OpenStruct.new({ id => '', method => (include_blank.is_a?(TrueClass) ? nil : include_blank) }))`
* for one line there's too much operations and event statement modifier at the end, in this case I'd rather split it like this

```ruby
if include_blank
  blank_option = OpenStruct.new({ id => '', method => (include_blank.is_a?(TrueClass) ? nil : include_blank) }
  array.insert(0, blank_option)
end
```

now it might be a good time to extract it to it's own method and use it when calling `f.colection_select` :-)